src/components/NavBar/NavBarLogo.tsx

```diff
@@ -1,21 +1,16 @@
 import { Link } from 'react-router-dom';
 import OfekLogo from '../../assets/images/OfekLogo.png';
+import LogoStyle from './Logo.module.css';
 
 interface LogoPropsType {
   header: string;
 }
 const NavBarLogo = ({ header }: LogoPropsType) => (
-  <nav>
-    <div className="dropdown-menu">
-      <div className="dropdown-menu-item">
-        <div className="dropdown-container">
-          <Link to="/">
-            <img src={OfekLogo} alt="ofek unit logo" className="logo-icon" />
-            <span className="logo-text">{header}</span>
-          </Link>
-        </div>
-      </div>
-    </div>
+  <nav className={LogoStyle.logoContainer}>
```

Comment:
you sure that every time you create **part** of the navbar you need to put <nav> tags around it?